sdk/search/azure-search-documents/azure/search/documents/_internal/_search_client.py

```diff
@@ -100,6 +101,15 @@ def close(self):
         """
         return self._client.close()
 
+    def get_search_index_buffered_sender(self, **kwargs):
```

Comment:
The "client" name is `SearchIndexingBufferedSender`, so this should be `get_search_indexing_buffered_sender`; however, in .NET we actually use the `Create` method name instead of `Get`.

I wonder, though, the `SearchIndexClient` has a `GetSearchClient` method, so maybe .NET really should be `GetSearchIndexingBufferedSender`. @tg-msft?

Reply:
That's a good question.  We should bring that up in ApiView and the next Arch Board meeting.  I went with `Create` because it's building a mutable instance instead of getting a reference to a resource, but `Create` is typically reserved for service calls.  (We also called in `BlobBatchClient.CreateBatch` in Storage for the same reason.)

Reply:
https://apiview.dev/Assemblies/Review/d40a51d26cf2444a85d302758ca3a2d1/8019df8163f94f24a03ad4f96577e6f1?doc=False#2035caadf2ed40bb908aef0e88c902e4